controller/rest/registry.go

```diff
@@ -153,23 +153,75 @@ func parseFilter(filters []string, regType string) ([]*share.CLUSRegistryFilter,
 	return repoFilters, nil
 }
 
-func handlerRegistryCreate(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
-	log.WithFields(log.Fields{"URL": r.URL.String()}).Debug("")
-	defer r.Body.Close()
+func RegistryConfigV2ToV1(v2data api.RESTRegistryConfigDataV2) api.RESTRegistryConfigData {
```

Comment:
this function is not exported by rest package.
its function name should be registryConfigV2ToV1, not RegistryConfigV2ToV1